tests/auxil/deprecations.py

```diff
@@ -72,3 +72,16 @@ def check_thumb_deprecation_warnings_for_args_and_attrs(
         )
 
     return True
+
+
+def check_thumb_deprececation_warning_for_method_args(
```

Comment:
```suggestion
def check_thumb_deprecation_warning_for_method_args(
```
+ all usages.
I know that's a pun on @Poolitzer 's depreciation 😉